apps/userauth/views.py

```diff
@@ -80,56 +81,59 @@ def post(self, request: WSGIRequest) -> Union[HttpResponse, HttpResponseRedirect
         form = CustomUserAddDataForm(request.POST)
         if current_user.year_of_birth is not None or current_user.post_code is not None:
             return HttpResponse(
-                "You cannot change these values yourself once they are set. Instead, make a request to the administrators via the profile edit page."
+                "You cannot change these values yourself once they are set. "
+                "Instead, make a request to the administrators via the profile edit page."
```

Comment:
I think I just learnt a new python syntax, two strings next to each other get concatenated:

```py
>>> a = "foo" "bar"
>>> a
'foobar'
```

Didn't know that!